TrackingTools/GsfTools/interface/MultiGaussianState.icc

```diff
@@ -1,3 +1,5 @@
+#ifndef TrackingTools_GsfTools_MultiGaussianState_icc
+#define TrackingTools_GsfTools_MultiGaussianState_icc
 #include "TrackingTools/GsfTools/interface/MultiGaussianState.h"
 #include "TrackingTools/GsfTools/interface/MultiGaussianStateCombiner.h"
```

Comment:
wouldn't it be more practical or at least less recursive-looking to simply merge MultiGaussianState.h, MultiGaussianState.icc, MultiGaussianStateCombiner.h, and MultiGaussianStateCombiner.icc
?

The updates made in this PR are adding a touch of duality on top what's already looking recursive.

Reply:
Merging them should work. I was trying to do the absolute minimum change since I do not know the original author's intent.